lib/ui/text.dart

```diff
@@ -598,19 +598,15 @@ class ParagraphStyle {
   ///   dropped. The width constraints are those set in the
   ///   [ParagraphConstraints] object passed to the [Paragraph.layout] method.
   ///
-  /// * `fontFamily`: The name of the font to use when painting the text (e.g.,
-  ///   Roboto).
+  /// * `fontFamily`: The fallback name of the font to use when painting the text
```

Comment:
is "fallback" necessary here?

Reply:
This is not used unless the there is no textstyle attached to the span.

Reply:
I wonder if it might be clearer to state this as: `The name of the font to apply when painting the text, in the absence of a ``textStyle`` being attached to the span.`

Reply:
That sounds great!